InstallDebian.md

```diff
@@ -0,0 +1,83 @@
+# TDS Foreign data wrapper
+
+* **Author:** Geoff Montee
+* **Name:** tds_fdw
+* **File:** tds_fdw/InstallDebian.md
+
+## Installing on Debian
+
+This document will show how to install tds_fdw on Debian 10. Other Debian distributions should be similar.
+
+### Install FreeTDS
+
+The TDS foreign data wrapper requires a library that implements the DB-Library interface,
+such as [FreeTDS](http://www.freetds.org).
+
+```bash
+sudo apt-get install libsybdb5 freetds-dev freetds-common
+```
+
+### Install PostgreSQL
+
+If you need to install PostgreSQL, do so by following the [apt installation directions](https://wiki.postgresql.org/wiki/Apt). For example, to install PostgreSQL 11 on Debian:
+
+```bash
+sudo bash -c "echo \"deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -c -s)-pgdg main\" > /etc/apt/sources.list.d/pgdg.list"
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install postgresql-11 postgresql-client-11 postgresql-server-dev-11
+```
+
+#### Important
+
+If you already have PostgreSQL installed on your system be sure that the package postgresql-server-dev-XX is installed too (where XX stands for your PostgreSQL version). 
+
+
+### Install tds_fdw
+
+#### Build from release package
+
+With Debian 10 it's recommended to use one of the release packages.
+You can download and install them via something like the following:
+
+```bash
+wget https://github.com/tds-fdw/tds_fdw/archive/v2.0.0-alpha.3.tar.gz
+tar -xvzf v2.0.0-alpha.3.tar.gz
+cd tds_fdw-2.0.0-alpha.3/
+make USE_PGXS=1
```

Comment:
We should also mention that `gcc` and `make` are also needed.

Reply:
Hi Julio,
sorry I gave the presence of those packages for granted.

Good suggestions, I will try to find the time to add mention to the various required packages asap.

How should I proceed?
Forking again the main repo, branching for a new "feature" and then submitting pr again?
Or I can keep my current fork, updating it with a pull and then branching and submitting pr?

Reply:
@MarzoLinus don't worry, I already fixed it at https://github.com/tds-fdw/tds_fdw/pull/232 can you have a look and approve?

My suggestion about how to handle a future PR (it's how I do it, but there are other alternatives) would be:

a) Add a new remote to your local clone repository, and call it `upstream` (CLI call: `git remote add upstream https://github.com/tds-fdw/tds_fdw.git`)
b) That way you can sync `master` using `upstream` (CLI call: `git checkout master && git pull upstream master`, the first commands ensures you are at master, the second gets the changes).
c) Create a new feature branch, do the changes, commit and push.
d) Create a PR.
e) When the PR is merged, or when you want to work on new features or bugfixes, start with step `b`, syncing `master` against `upstream` and follow the rest of steps.

For all the steps above you can use a Git GUI client, of course, but I only work with Git via CLI, so I am not familiar with GUI clients :-)

BTW: Can you also have a look at https://github.com/tds-fdw/tds_fdw/issues/230 ? I was unable to reproduce your problems with Debian 10.